ui/src/app/shared/header/header.component.spec.ts

```diff
@@ -12,17 +12,33 @@ import {
   MatListModule,
   MatMenuModule,
   MatNativeDateModule,
+  MatPaginatorModule,
 } from "@angular/material";
 import {FormsModule, ReactiveFormsModule} from "@angular/forms";
 import {RouterTestingModule} from "@angular/router/testing";
 import {BrowserAnimationsModule} from "@angular/platform-browser/animations";
+import {BehaviorSubject} from 'rxjs/BehaviorSubject';
 
+import {JobListView} from "../job-stream";
 import {HeaderComponent} from "./header.component";
 import {startCol, statusesCol} from "../common";
+import {QueryJobsResult} from '../model/QueryJobsResult';
 import {JobStatus} from "../model/JobStatus";
 
 
 describe('HeaderComponent', () => {
```

Comment:
+ a test for should show/hide the paginator depending on jobs present

Reply:
Done, along with other tests.